.evergreen/config.yml

```diff
@@ -299,8 +299,10 @@ functions:
           STORAGE_ENGINE: ${STORAGE_ENGINE}
           LOAD_BALANCER: ${LOAD_BALANCER}
           REQUIRE_API_VERSION: ${REQUIRE_API_VERSION}
+          LOCAL_ATLAS: ${LOCAL_ATLAS}
```

Comment:
Note: `include_expansions_in_env` is preferable to `env` for `X: ${X}` (definition used as-is).

Reply:
Updated